### PR TITLE
Favorited services

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://issues.redhat.com/browse/RHCLOUD-23669

Screen Shot 2023-01-17 at 11 45 05 AM
Screen Shot 2023-01-17 at 11 45 30 AM

### CODE REVIEW

**Comment by maryshak1996** [comment]
@epwinchell this is looking awesome! My only question -- why is 'View all services' button disabled in the empty state? We want them to take that action to add things to their favorites

**Comment by epwinchell** [comment]
@maryshak1996 @Hyperkid123 will activate it in his PR.

**Comment by Hyperkid123** [comment]
As long as the favorites button in the masthead stays disabled before we have the feature enabled in the prod backend, feel free to do anything you like on this page. :)

File: src/components/Header/FavoritesLink.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import ChromeLink from '../ChromeLink';
 
 const FavoritesLink = () => {
   return (
-    <Button className="chr-c-button-masthead disabled" component={(props) => <ChromeLink {...props} href="#" />}>
+    <Button className="chr-c-button-masthead" component={(props) => <ChromeLink {...props} href="/FavoritedServices" />}>
```
Comment (Hyperkid123) [inline src/components/Header/FavoritesLink.tsx:9]: Can we disable the link for now? Until the backend is hooked up no favoriting will work anyways. And we should also use the `isDisabled` prop on the button now that the link has a pathname.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**